### PR TITLE
Add a docker image using debian

### PR DESCRIPTION
**Pull request by maraino**
### Description

This commit adds a workflow to build a docker image using Debian bullseye. The image format will be like this:
 - smallstep/step-cli:bullseye
 - smallstep/step-cli:0.24.0-bullseye

### CODE REVIEW

**Review by tashian — APPROVED** [review]
lgtm

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -21,11 +21,13 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKER_IMAGE: smallstep/step-cli
+      DEBIAN_VERSION: bullseye
     outputs:
       version: ${{ steps.extract-tag.outputs.VERSION }}
       vversion: ${{ steps.extract-tag.outputs.VVERSION }}
       is_prerelease: ${{ steps.is_prerelease.outputs.IS_PRERELEASE }}
       docker_tags: ${{ env.DOCKER_TAGS }}
+      docker_debian_tags: ${{ env.DOCKER_DEBIAN_TAGS }}
```
Comment (tashian) [inline .github/workflows/release.yml:30]: you could make this docker_tags_debian to align with step-kms-plugin docker_tags_cloud

Comment (maraino) [reply .github/workflows/release.yml]: Fixed with 9bd605c

**Review by tashian — APPROVED** [review]
lgtm

**Review by dopey — APPROVED** [review]

**Pull request merged**